File: backend/pkg/testdata/gfmt/subtasks-groups/solutions/ok.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    if (cin >> n) {
        if (n % 2 == 0) {
            cout << "Even" << endl;
        } else {
            cout << "Odd" << endl;
        }
    }
    return 0;
}

File: backend/internal/templates/testdata/subtasks-groups/solutions/ok.py
import sys

def main():
    line = sys.stdin.read().strip()
    if not line:
        return
    n = int(line)
    if n % 2 == 0:
        print("Even")
    else:
        print("Odd")

if __name__ == '__main__':
    main()
